cmd/common.go

```diff
@@ -61,13 +66,18 @@ func assertInFlutterProject() string {
 			goto Fail
 		}
 
-		return pubspec.Name
+		if pubspec.Author == "" {
+			fmt.Println("hover: Missing `author` in pubspec.yaml.")
+			goto Fail
+		}
+
+		return pubspec
 	}
 
 Fail:
 	fmt.Println("hover: This command should be run from the root of your Flutter project.")
```

Comment:
When coming here from, for example, parsing error on pubspec.yaml, this error may well be a false flag. The pubspec couldn't be parsed, but we may well be in the root of a valid flutter project.